projects/onto-search/src/lib/onto-search-field/onto-search-field.component.ts

```diff
@@ -2,24 +2,50 @@ import {Component, EventEmitter, Input, OnInit, Output, TemplateRef} from '@angu
 import {Observable, of} from 'rxjs';
 import {SearchModel} from './models/search-model';
 
+
 @Component({
   selector: 'onto-search-field',
   templateUrl: './onto-search-field.component.html',
   styleUrls: ['./onto-search-field.component.scss'],
 })
 export class OntoSearchFieldComponent implements OnInit {
+  /**
+   * Custom template reference. Will override the template used in autocomplete dropdown.
+   */
   @Input()
   customTemplate?: TemplateRef<any>;
+
+  /**
+   * Preselected states list.
+   */
   @Input()
   private selectedList: any[];
+
+  /**
+   * Autocomplete response
+   */
   @Input()
   private autocompleteData: Observable<SearchModel[]>;
+
+  /**
+   * Configuration used to enable or disable inner autocomplete filtration. False by default if not passed.
```

Comment:
Maybe it's by me, but I just don't get it what's the purpose of this flag. And the comment doesn't say much more than the obvious. I'm just not too familiar with these material ui components.